src/scripts/release_picard.sh

```diff
@@ -184,12 +184,12 @@ cd $TMPDIR
 # clone
 git clone $PICARDGITROOT picard
 cd picard
-ant clone-htsjdk
+ant -buildfile build-from-src.xml clone-htsjdk
```

Comment:
It is possible. Just a little work. Alternately we could just ensure that the builds have been tested and released to OSS. And **then** do the release to github. I think for simplicity's sake we really should only be releasing to one place (OSS) and the github release script could just become a documentation update.